custom_components/ramses_cc/binary_sensor.py

```diff
@@ -125,7 +125,8 @@ def available(self) -> bool:
     @property
     def is_on(self) -> bool | None:
         """Return the state of the binary sensor."""
-        return getattr(self._device, self.entity_description.attr)
+        result = getattr(self._device, self.entity_description.attr)
+        return None if result is None else bool(result)
```

Comment:
I don't think we need to do this - I checked, and all of these attrs appear to be `bool`s.

Reply:
This was just so that the override in `RamsesLogbookBinarySensor` could be removed but now that I think about it I don't think that override is quite what we want.